geode-core/src/main/java/org/apache/geode/internal/cache/MemberFunctionStreamingMessage.java

```diff
@@ -188,7 +190,7 @@ protected void process(final ClusterDistributionManager dm) {
       FunctionContextImpl context = new MultiRegionFunctionContextImpl(cache,
           this.functionObject.getId(), this.args, resultSender, regions, isReExecute);
 
-      long start = stats.startTime();
+      start = stats.getTime();
```

Comment:
I think i'm just confused, but I think you are suggesting we can remove this getTime() call here as long as we are ok with adding the context creation etc to the total execution time?  I think I would agree.

Reply:
Yes, that was one of the things I/we suggested. Do you know if users would care about this change to the total execution time?

Reply:
Perhaps we can put a post on user and dev lists explaining the change and see if anyone objects?  I can't think of a reason other than maybe some performance test complaining somewhere (assuming it used stats).

Reply:
I am going to check with the user/dev list.